src/cpp/include/reporting/logger.h

```diff
@@ -26,30 +26,30 @@ class MariusLogger {
   public:
     shared_ptr<spdlog::logger> main_logger_;
 
-    MariusLogger(string base_directory) {
+    MariusLogger(string dataset_dir) {
```

Comment:
Does it make sense to keep the logs in the dataset directory? It might be better to have this be the model directory, otherwise if multiple experiments are using the same dataset the logs will be overwritten.